index.html

```diff
@@ -179,45 +193,55 @@ <h3 class="titlePost">Multi-Post Stories</h3>
-                            alt="github"></a></li>
-
-                <li><a class="linkedin flex" role="button" href=""><img class="comunic"
-                            src="icons&imgs/Icon-Linkedin.png" alt="linkedin"></a></li>
-
-                <li><a class="angelist flex" role="button" href=""><img class="comunic" src="icons&imgs/angelist.png"
-                            alt="angelist"></a></li>
-
-                <li><a class="twitter flex" role="button" href=""><img class="comunic" src="icons&imgs/twitter.png"
-                            alt="twitter"></a></li>
-
-                <li><a class="medium flex" role="button" href=""><img class="comunic" src="icons&imgs/Vector.png"
-                            alt="medium"></a></li>
-            </ul>
-            <h5 class="e-address">moathalsabahKachi@gmail.com</h5>
-            </footer>
-        </div>
+            <form action="https://formspree.io/f/xpzbgqjj" method="POST" id="contact-form" class="flex column">
+                <div class="desk flex">
+                    <input name="fname" id="fname" type="text" class="field1" placeholder="First name" maxlength="30"
+                        required>
+                    <input name="lname" id="lname" type="text" class="field1" placeholder="Last name" maxlength="30"
+                        required>
+                </div>
+                <input name="name" id="name" type="text" class="field1" placeholder="Text Input" maxlength="30"
+                    required>
+                <input class="field2" name="email" id="email" type="email" placeholder="Examle@gmail.com" required>
+                <textarea class="field3" name="message" placeholder="Enter text here" maxlength="500"
+                    required></textarea>
+                <button type="submit" class="button normal email">Send</button>
+            </form>
+            <div class="reach">
+                <ul class="socialMedia flex">
+                    <li><a class="github flex" role="button" href=""><img class="comunic"
+                                src="icons&imgs/Icon-GitHub.png" alt="github"></a></li>
+
+                    <li><a class="linkedin flex" role="button" href=""><img class="comunic"
+                                src="icons&imgs/Icon-Linkedin.png" alt="linkedin"></a></li>
+
+                    <li><a class="angelist flex" role="button" href=""><img class="comunic"
+                                src="icons&imgs/angelist.png" alt="angelist"></a></li>
+
+                    <li><a class="twitter flex" role="button" href=""><img class="comunic" src="icons&imgs/twitter.png"
+                                alt="twitter"></a></li>
+
+                    <li><a class="medium flex" role="button" href=""><img class="comunic" src="icons&imgs/Vector.png"
+                                alt="medium"></a></li>
+                </ul>
+                <h5 class="e-address">moathalsabahKachi@gmail.com</h5>
```

Comment:
- [ ] like the previous reviewer noted, please add your submit buttons margin top and button, so it takes after the figma design 
- [ ] you have done a great job implementing all the previous reviewer's requested changes 😄 . Kindly add the social icons, so it can help as links to your different social sites. it's like you forgot to do that.
Your version
![Screenshot (232)_LI](https://user-images.githubusercontent.com/65298048/177930907-35c16d7c-5cb9-4018-9170-b198d4978e5b.jpg)
figma version
![Screenshot (233)_LI](https://user-images.githubusercontent.com/65298048/177931033-853ac24d-7c34-4444-ac59-44b37ccc003c.jpg)

Reply:
They do show up for me the have never disappeared. but margin bottom and top I am going to add them now. 😃